package.json

```diff
@@ -50,17 +56,19 @@
     "platform": "1.3.5"
   },
   "devDependencies": {
+    "@types/connect": "^3.4.32",
+    "@types/jest": "^24.0.13",
+    "@types/platform": "1.3.2",
+    "@types/supertest": "^2.0.7",
+    "@typescript-eslint/eslint-plugin": "^1.9.0",
+    "@typescript-eslint/parser": "^1.9.0",
+    "connect": "^3.7.0",
     "content-security-policy-parser": "^0.2.0",
-    "express": "^4.17.1",
-    "mocha": "^6.2.0",
-    "standard": "^13.1.0",
-    "supertest": "^4.0.2"
-  },
-  "standard": {
-    "globals": [
-      "describe",
-      "beforeEach",
-      "it"
-    ]
+    "eslint": "^5.16.0",
```

Comment:
Any reason this isn't using the latest ESLint?

Reply:
Have left at the latest ESLint `5.x` for now - there are some breaking changes in `6.x` so am not sure yet if that's compatible with `eslint-config-helmet` as it stands. 🤔